nowcasting_dataset/data_sources/fake/batch.py

```diff
@@ -553,39 +553,41 @@ def create_image_array(
         size_x=image_size_pixels_width,
         x_center_osgb=x_center_osgb,
         y_center_osgb=y_center_osgb,
+        two_dimensional=nwp_or_satellite == "satellite",
```

Comment:
what about hrv?

Reply:
maybe we dont need to worry about that

Reply:
Perhaps it would be better to do it as 
```
two_dimensional = Optional[bool] = None

if two_dimensional is None:
      two_dimensional=nwp_or_satellite == "satellite"
```

Just seems a bit confusing in the function inputs